packages/clay-navigation/src/BreadcrumbEllipsis.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import ClayButton from '@clayui/button';
+import ClayDropDown from '@clayui/drop-down';
+import ClayIcon from '@clayui/icon';
+import React, {useState} from 'react';
+import {IBreadcrumbItem} from './BreadcrumbItem';
+
+export interface IBreadcrumbEllipsisProps {
+	items?: Array<IBreadcrumbItem>;
+
+	spritemap: string;
+}
+
+export const BreadcrumbEllipsis: React.FunctionComponent<
+	IBreadcrumbEllipsisProps
+> = ({items, spritemap, ...otherProps}) => {
+	const [active, setActive] = useState(false);
+	return (
```

Comment:
line break above this. Essentially we put line breaks where "order is important", so if we move the return above the `const`, it'd break.

Reply:
I didn't know that. Thanks for the tip!